front/src/modules/trainschedule/components/ManageTrainSchedule/hooks/useUpdateTimetableItem.ts

```diff
@@ -78,39 +69,30 @@ const useUpdateTimetableItem = (
     setIsWorking(true);
 
     let trainIdToSelect: TrainId | undefined;
-
-    // handle item update without changing type
     if (
       !showPacedTrains ||
       formattedSimulationConf.editingTrainIsPacedTrain === isPacedTrain(trainIdToEdit)
     ) {
```

Comment:
Hum I think you can even simplify the code like this:
```suggestion
    if (formattedSimulationConf.editingTrainIsPacedTrain) {
      const updatedItem = await storePacedTrain(
        trainIdToEdit,
        formatPacedTrainPayload(formattedSimulationConf),
        timetableId!,
        dispatch,
        upsertTimetableItems,
        removeTimetableItems
      );

      // if the selected item is an occurrence of the edited paced train, keep it selected
      // else select the first occurrence by default
      trainIdToSelect =
        selectedTrainId &&
        isOccurrence(selectedTrainId) &&
        extractPacedTrainIdFromOccurrenceId(selectedTrainId) === trainIdToEdit
          ? selectedTrainId
          : formatPacedTrainIdToOccurrenceId(updatedItem.id, 0);
    } else {
      const updatedItem = await storeTrainSchedule(
        trainIdToEdit,
        formatTrainSchedulePayload(formattedSimulationConf, confName, startTime),
        timetableId!,
        dispatch,
        upsertTimetableItems,
        removeTimetableItems
      );
      trainIdToSelect = updatedItem.id;
    }
```

Reply:
True, the `showPaced` was redundant. It is not way clearer, thanks

Done in "front: simplify useUpdateTimetableItem"